src/CSET/_workflow_utils/finish_website.py

```diff
@@ -2,14 +2,58 @@
 
 """Write finished status to website front page.
 
-Does the final update to the workflow status on the front page of the web
-interface.
+Constructs the plot index, and does the final update to the workflow status on
+the front page of the web interface.
 """
 
+import json
+import logging
 import os
+from pathlib import Path
 
+from CSET._common import combine_dicts
 
-def run():
-    """Run workflow script."""
-    with open(f"{os.getenv('WEB_DIR')}/status.html", "wt") as fp:
+logging.basicConfig(
+    level=os.getenv("LOGLEVEL", "INFO"), format="%(asctime)s %(levelname)s %(message)s"
+)
+
+
+def construct_index():
+    """Construct the plot index.
+
+    Index should has the form {"Category Name": {"recipe_id": "Plot Name"}}
```

Comment:
Think it would be useful to say what `recipe_id` is in the docstring.

Reply:
Clarified in 86df5b278b250efaf8331617ff8267baf0af4f82